docker-compose-net/docker-compose-net.md

```diff
@@ -0,0 +1,234 @@
+---
+layout: guides
+title: How to Use Docker Compose with Weave Net
+---
+
+Docker Compose is used to configure multiple container apps. A Docker Compose file configures your application's services and with a single command creates and starts services. Depending on how you are using Weave Net, whether through the Docker API Proxy or by using the Weave Docker Plugin, the syntax of the Docker Compose file differs. This guide provides an overview of the different ways in which networking can be configured and also provides examples of the Docker Compose files for Weave Net. 
+
+The following topics are discussed: 
+
+  * [What You Will Use](#what-use)
+  * [Before You Begin](#before)
+  * [Attaching Containers to a Weave Network](#attaching-containers)
+  * [Docker Compose and the Weave Net Docker API Proxy](#setup-proxy)
+     * [Docker Compose File Syntax for the Weave Docker API Proxy](#compose-proxy)
+  * [Docker Compose and the Weave Net Docker Plugin](#setup-plugin)
+    * [Docker Compose File Syntax and the Weave Net Docker Plugin](#compose-plugin)
+
+##<a name="what-use"></a>What You Will Use
+
+* [Weave](http://weave.works)
+* [Weave Scope](http://weave.works/scope/index.html)
+* [Docker](http://docker.com)
+* [Docker Compose](https://www.docker.com/docker-compose)
+
+##<a name="before"></a>Before You Begin
+
+If you are using Windows, you can install [Docker for Windows](https://docs.docker.com/engine/installation/windows/), and if you are on a Mac, install [Docker for Mac](https://docs.docker.com/engine/installation/mac/) which provides all of the tools you need to complete this guide.
```

Comment:
On second thoughts, this guide is tightly coupled to `docker-machine`, it won't work at all on Docker for Mac because you can't create multiple VMs on Docker for Mac. I presume the same is true of Windows. So let's scrap this paragraph and just insist that users install `docker-machine`.